### DISCUSSION TITLE
Tag Created Webhook Payload Missing Project Metadata

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#413] [Product Feedback]
### What product would you like to leave feedback for?

Console

### What Console version are you using?

12

### Description

Hi,

The documentation for the [Tag Created Webhook](https://docs.mia-platform.eu/docs/development_suite/webhooks-and-events/events#tag-created) mentions that the payload of the webhook call should contain a `project` object field. However, after trying it, it seems that this property is not included in the payload of the webhook call. Moreover, the Table shown on the documentation is incoherent with the JSON Schema for the webhook payload shown in the documentation, as this property is not marked as required in the JSON schema (see attached pictures)
![image](https://github.com/mia-platform/community/assets/21307773/21e2c938-7524-4757-90fd-ae50b65ddf13)
![image](https://github.com/mia-platform/community/assets/21307773/382c4dce-8c39-4e12-a1e2-702e670fd8ae)

Is this a bug of the Console, or is the documentation misleading? If the latter, I would suggest including the information about the project in this webhook payload, since it would enable different workflows.

Thank you,

Giorgio

**Comment by mattia-fumo** [comment]
The documentation is correct. It'a a known bug and yet to be fixed, thank you for reporting it.

**Reply by GioAc96** [reply]
Thank you for the clarification. Besides fixing the bug, I would also suggest to update the documentation by making the JSON schema coherent witht the Event Payload table.

**Comment by alexevansvu** [comment]
This appears to still be an issue as of Mia-Platform Version 13.0.2 / Console Version: 1.102.8

Is fixing this bug on any current roadmaps?

**Comment by malta895** [comment]
Hello @GioAc96 and @alexevansvu,
thanks again for your feedback.
We confirm the issue has been addressed and a fix will be included in the next Console release `v13.2.1`, expected to be released on the 24th of October.

Thank you
